plugins/workspace-backpack/src/backpack.ts

```diff
@@ -534,7 +535,7 @@ export class Backpack
    * @param block The block to be added to the backpack.
    */
   addBlock(block: Blockly.Block) {
```

Comment:
should these methods be updated to take a `Backpackable` object instead of block? I wouldn't change the name, just the type of param and jsdoc

Reply:
The `Blockly.Block` doesn't implement `Backpackable`, only `Blockly.BlockSvg` does, because backpacks don't exist in headless mode. So changing this to take in a `Backpackable` would be a breaking change.

Which actually means I implemented this incorrectly, whoops. If it's not a `Backpackable` we should use the old codepath.